util/app-config/src/configs/store.rs

```diff
@@ -15,6 +15,13 @@ pub struct Config {
     pub block_uncles_cache_size: usize,
     /// TODO(doc): @doitian
     pub cellbase_cache_size: usize,
+    /// whether enable freezer
+    #[serde(default = "default_freezer_enable")]
+    pub freezer_enable: bool,
```

Comment:
Just an advice: if we may add more options to freezer, may be a standalone section is better.
For example:  `path`, `max_open_files_limit`, `max_freeze_limit`, `threshold_epoch` and so on.